ecl/eclagent/eclagent.cpp

```diff
@@ -2502,6 +2502,8 @@ bool EclAgent::checkPersistUptoDate(IRuntimeWorkflowItem & item, const char * lo
         errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
```

Comment:
@jamienoss I got it now.  I didn't look at the code carefully enough. Thanks.  @jakesmith can you comment on jamie's question - any problems in using the file if CRC is not present?